.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,149 @@
+      $ system_profiler -detailLevel mini SPSoftwareDataType | head -n 6
+      Software:
+
+          System Software Overview:
+
+            System Version: macOS 10.15.7 (19H1323)
+            Kernel Version: Darwin 19.6.0
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Steps to Reproduce
+    description: >-
+      Describe exactly how to reproduce the problem, using a minimal test-case.
+      It would *really* help us understand your problem if you paste in the Python code
+      that you're running.
+
+
+      **HINT:** You can paste [GitHub Gist](https://gist.github.com) links for larger files.
+    value: |
+      <!--- Paste your minimal failing Python example code between the quotes below -->
+      ```python (paste below)
+
+      ```
+
+      <!--- ...or if you have a failing CLI command paste it between the quotes below -->
+      ```console (paste below)
+
+      ```
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: File Upload (optional)
+    description: >-
+      If your steps to reproduce your minimal failing example require either a spec or a
+      workspace file, please upload it by attaching it to the text area here.
+
+
+      **HINT:** You can paste [GitHub Gist](https://gist.github.com) links for larger files.
+    placeholder: >-
+      Attach any files or compressed archives by dragging & dropping, selecting,
+      or pasting them here.
+  validations:
+    required: false
+
+- type: textarea
+  attributes:
+    label: Expected Results
+    description: >-
+      Describe what you expected to happen when running the steps above.
+    placeholder: >-
+      I expected X to happen because I assumed Y.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Actual Results
+    description: >-
+      Paste verbatim program or command output.
+      Don't wrap it with triple backticks &mdash; your whole input will be
+      turned into a code snippet automatically.
+    render: console
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Cookiecutter Version
+    description: >-
+      Paste verbatim output from `Cookiecutter --version` below, under the prompt line.
+      Don't wrap it with triple backticks &mdash; your whole input will be
+      turned into a code snippet automatically.
+    render: console
+    placeholder: |
+      $ Cookiecutter --version
+  validations:
+    required: true
+
+- type: checkboxes
+  attributes:
+    label: Code of Conduct
+    description: |
+      Read the [`Cookiecutter` Code of Conduct][CoC] first.
+
+      [CoC]: https://github.com/osl-incubator/cookiecutter-python/blob/main/CODE_OF_CONDUCT.md
+    options:
+    - label: I agree to follow the Code of Conduct
+      required: true
+...
+Thanks [Pyhf](https://github.com/scikit-hep/pyhf/tree/main/.github/ISSUE_TEMPLATE) for  for coming up with such detailed and great templates slightly_smiling_face
```

Maybe something like this?
```suggestion
This template has been adopted from [pyhf](https://github.com/scikit-hep/pyhf/tree/main/.github/ISSUE_TEMPLATE)'s excellent bug report template.
```